apps/crn-server/test/handlers/research-output/algolia-index-research-output-handler.test.ts

```diff
@@ -30,7 +30,25 @@ describe('Research Output index handler', () => {
     await indexHandler(createEvent('ro-1234'));
 
     expect(algoliaSearchClientMock.save).toHaveBeenCalledWith({
-      data: researchOutputResponse,
+      data: expect.objectContaining(researchOutputResponse),
+      type: 'research-output',
+    });
+  });
+
+  test('Should populate _tags field before saving the research-output to Algolia', async () => {
```

Comment:
I think it's best if you explicitly declare a tag on each of the properties (environments, subtype, etc) to make it easier to understand (and more future proof).